### PR TITLE
AVT-1025: Verify toast notification when Persist Notification is ON

### PR DESCRIPTION
**Pull request by tranducanh**
## Description
Verify toast notification when Persist Notification is ON  [AVT-1025](https://perzoinc.atlassian.net/browse/AVT-1025)

## Run
Please refer attachment for more details
[Jest Test Report_AVT_1025.zip](https://github.com/symphonyoss/SymphonyElectron/files/2248568/Jest.Test.Report_AVT_1025.zip)

## QA Checklist
- [] Unit-Tests
- [x] Automation-Tests

### CODE REVIEW

**Comment by vantruongt2** [comment]
New result 

[Jest Test Report.zip](https://github.com/symphonyoss/SymphonyElectron/files/2256650/Jest.Test.Report.zip)

File: tests/spectron/spectronConstants.js [hunk 1]
```diff
@@ -7,8 +7,7 @@ module.exports = {
 
     SEARCH_LIBRARY_PATH_MAC: "node_modules/electron/dist/Electron.app/Contents/library",
     SEARCH_LIBRARY_PATH_WIN: "node_modules/electron/dist/library",
-    TESTED_HOST: "https://cip4-qa.symphony.com/",
-
+    TESTED_HOST: "https://qa4.symphony.com",
```
Comment (VishwasShashidhar) [inline tests/spectron/spectronConstants.js]: do we want to hard code this?

File: tests/spectron/spectronConstants.js [hunk 2]
```diff
@@ -20,5 +19,10 @@ module.exports = {
         }
     },
 
-    LOG_FILENAME_PREFIX: "logs_symphony_"
+    LOG_FILENAME_PREFIX: "logs_symphony_",
+
+    USER_A: {username:"jest01",password:"Symphony!123456",name:"jest01 jest01"},
```
Comment (VishwasShashidhar) [inline tests/spectron/spectronConstants.js]: do we want to hard code this?

File: tests/spectron/spectronConstants.js [hunk 3]
```diff
@@ -20,5 +19,10 @@ module.exports = {
...
+    USER_B: {username:"jest02",password:"Symphony!123456",name:"jest02 jest02"},
+    USER_C: {username:"jest03",password:"Symphony!123456"},
```
Comment (KiranNiranjan) [inline tests/spectron/spectronConstants.js]: I don't think you can store passwords here.

**Pull request merged**